app/__init__.py

```diff
@@ -1,7 +1,32 @@
 from flask import Flask
+from flask_sqlalchemy import SQLAlchemy
+from flask_migrate import Migrate
+from dotenv import load_dotenv
+import os
 
+db = SQLAlchemy()
+migrate = Migrate()
+load_dotenv()
 
 def create_app(test_config=None):
     app = Flask(__name__)
 
+
+    if not test_config:
+        app.config['SQLALCHEMY_TRACK_MODIFICATIONS'] = False
+        app.config['SQLALCHEMY_DATABASE_URI'] = \
+        'postgresql+psycopg2://postgres:postgres@localhost:5432/solar_system_development'
```

Comment:
Just a note:  In future projects we can use an environmental variable rather than hard-coding the connection string here, like you do below with `os.environ.get(
            "SQLALCHEMY_TEST_DATABASE_URI")`